src/components/wizard-upload-configfile/wizard-upload-configfile.html

```diff
@@ -0,0 +1,21 @@
+<div class="col-md-6">
+  <div class="form-group has-feedback"
+       ng-class="formFeedback('{{$ctrl.details.tunnelType}}.{{$ctrl.details.property}}')">
+    <label for="{{$ctrl.details.tunnelType}}.{{$ctrl.details.property}}" class="col-md-5 control-label">
+      {{'wizard-upload-configfile.' + $ctrl.details.property + '.label' | translate}}
+    </label>
+    <load-file-button
+            id="{{$ctrl.details.tunnelType}}.{{$ctrl.details.property}}"
+            accept="{{$ctrl.details.fileExtensions}}"
+            base64="true"
+            on-loaded="$ctrl.uploadContent(content)"
+            required="{{$ctrl.details.required}}">
```

Comment:
without `{{}}` (unless you really want a string here which I doubt)

Reply:
you're right, it would be just true or false... done